### PR TITLE
Import FJC updates.

### PR DESCRIPTION
**Pull request by jonashleyo**
Updated to use FJC variable names and automatically include FJC as a source when uploading FJC CSVs.  Also a small tweak to date formatting (yyyy-mm-dd) so that they're read correctly when imported.

### CODE REVIEW

File: cl/people_db/import_judges/populate_fjc_judges.py [hunk 1]
```diff
@@ -189,17 +187,17 @@ def add_positions_from_row(item, person, testing, fix_nums=None):
             position_type='jud',
             date_start=date_start,
             date_termination=date_termination,
-            court_id=courtid,
+            court_id=courtid
```
Comment (mlissner) [inline cl/people_db/import_judges/populate_fjc_judges.py]: We should always have trailing commas. They make adding/removing parameters in the future cleaner. Not all languages allow this, but Python, happily, does.

File: cl/people_db/import_judges/populate_fjc_judges.py [hunk 2]
```diff
@@ -529,19 +539,6 @@ def make_federal_judge(item, testing=False):
         else:
             city, state, org = '', '', ''
 
-        position = Position(
```
Comment (mlissner) [inline cl/people_db/import_judges/populate_fjc_judges.py]: It seems weird to remove this. Doesn't removing this mean that `position` on line 547 isn't defined? Maybe I'm missing it?

File: cl/people_db/models.py [hunk 3]
```diff
@@ -1169,6 +1169,14 @@ class Source(models.Model):
         blank=True,
     )
 
+    def save(self, *args, **kwargs):
```
Comment (mlissner) [inline cl/people_db/models.py]: Neat, but why can't aliases have source info? We might wonder where we learned that Michael went by Mike, right?

File: cl/people_db/import_judges/populate_fjc_judges.py (hunk 1 above)
Comment (jonashleyo) [reply cl/people_db/import_judges/populate_fjc_judges.py]: I didn't know about trailing commas in Python and I'll slip these back in.  Very cool.

File: cl/people_db/import_judges/populate_fjc_judges.py (hunk 2 above)
Comment (jonashleyo) [reply cl/people_db/import_judges/populate_fjc_judges.py]: You're correct.  I'd gotten ahead of myself testing something else out and inadvertently deleted this when I shouldn't have.

File: cl/people_db/models.py (hunk 3 above)
Comment (jonashleyo) [reply cl/people_db/models.py]: Yep.  I'll update this as well.  Thanks!

**Comment by mlissner** [comment]
Please let me know when this is ready for my review again.

**Comment by jonashleyo** [comment]
This should be ready for review.

File: cl/people_db/models.py (hunk 3 above)
Comment (mlissner) [reply cl/people_db/models.py]: Do we need this function here for some reason if we remove the other validation rule that you had? Seems to me (correct me if I'm wrong), that this save method can be nixed too?

File: cl/people_db/import_judges/populate_fjc_judges.py (hunk 1 above)
Comment (mlissner) [reply cl/people_db/import_judges/populate_fjc_judges.py]: Yeah, they're great. You went a little overboard though. You might find it useful to read Google's Python format guide: https://google.github.io/styleguide/pyguide.html

But the one thing I'll mention here is that you should have:

    some_function(arg1,
                  arg2)

Not:

    some_function(arg1,
                  arg2,)

I could probably drum up a reason why the former is preferred, but I think mostly it's just a style thing. No need to go back and fix this though...just mentioning it for the future.

Really, the above should be:

    some_function(arg1, arg2)

In any case!

**Comment by jonashleyo** [comment]
Yes, it can be nixed.  Somehow I thought this was needed in order to save
the FJC URL but now I see it isn't necessary.  I'll update it.

**Comment by jonashleyo** [comment]
Got it.  Thanks!

**Comment by jonashleyo** [comment]
I just updated this.

**Comment by jonashleyo** [comment]
Checking in to see if this still needs work or if it can be approved.  Not quite sure how to proceed.  When this is approved I'll submit another pull request that depends upon the updated variable names reflected here.

File: cl/people_db/import_judges/populate_fjc_judges.py [hunk 4]
```diff
@@ -445,7 +450,9 @@ def make_federal_judge(item, testing=False):
         else:
             school_str = ''
 
-        schoolname = item['Name of School' + school_str]
+        school_str = ' (%s)' % schoolnum
```
Comment (mlissner) [inline cl/people_db/import_judges/populate_fjc_judges.py]: This is weird. Revise?

**Comment by mlissner** [comment]
Just the one comment above. The rest looks good. Thanks @jonashleyo

File: cl/people_db/import_judges/populate_fjc_judges.py (hunk 4 above)
Comment (jonashleyo) [reply cl/people_db/import_judges/populate_fjc_judges.py]: I'll pull out 448-452 in the next pull request - every field related to education has a number wrapped in a parenthetical (including "1") - but I'm not quite sure I see the issue here.  In the FJC data the variable is "Name of School (1)", "Name of School (2)" and so on.

Comment (mlissner) [reply cl/people_db/import_judges/populate_fjc_judges.py]: Just that lines 448-452 don't have any effect after the change you added line 453.

Comment (jonashleyo) [reply cl/people_db/import_judges/populate_fjc_judges.py]: Should I submit an updated pull request omitting 448-452 now?  They should not be included and I already have another pull request waiting in the wings after this is approved that removes these.  But I can do it now if you prefer.

Comment (mlissner) [reply cl/people_db/import_judges/populate_fjc_judges.py]: Nah, if it's already ready to go, I'll merge this and you can send that. Merging.

**Pull request merged**